### PR TITLE
Css question 1

### PR DESCRIPTION
**Pull request by Shaik-mohd-huzaifa**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/css-quiz.ts [hunk 1]
```diff
@@ -1241,6 +1241,17 @@ const cssQuiz = [
...
+  },
+  {
+    Question:
+      "In CSS, Flex is a shorthand property for which of the following properties?",
+    Answer: "flex-grow, flex-shrink, and flex-basis",
+    Distractor1: "justify-content, align-items, and align-content",
+    Distractor2: "flex-direction, flex-wrap, and flex-flow",
+    Distractor3: "flex-direction, flex-wrap, and flex-flow",
+    Explanation:
+      "Flex is a shorthand property for all these three properties: flex-grow, flex-shrink, and flex-basis.",
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1253]:
```suggestion
      "Flex is a shorthand property for these three properties: flex-grow, flex-shrink, and flex-basis.",
```

File: src/data/css-quiz.ts [hunk 2]
```diff
@@ -1241,6 +1241,17 @@ const cssQuiz = [
...
       "You can use border-style: groove; to specify groove border styles around HTML elements.",
     Link: "https://www.freecodecamp.org/news/css-border-style-and-html-code-examples/"
+  },
+  {
+    Question:
+      "In CSS, Flex is a shorthand property for which of the following properties?",
+    Answer: "flex-grow, flex-shrink, and flex-basis",
+    Distractor1: "justify-content, align-items, and align-content",
+    Distractor2: "flex-direction, flex-wrap, and flex-flow",
+    Distractor3: "flex-direction, flex-wrap, and flex-flow",
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1251]: we don't want two answers to be identical

```suggestion
    Distractor3: "flex-direction, order, and flex-reverse",
```

File: src/data/linux-quiz.ts [hunk 3]
```diff
@@ -1269,14 +1269,24 @@ const linuxQuiz = [
...
     Answer: "> filename.extension",
     Distractor1: "cd filename.extension",
     Distractor2: "ls filename.extension",
     Distractor3: "None of the above",
     Explanation:
       "> greater than operator is used to create a new file in Linux.",
     Link: "https://www.freecodecamp.org/news/learn-the-50-most-used-linux-terminal-commands/"
+  },
+  {
+    Question: "In Linux, How to end an if statement?",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1283]:
```suggestion
    Question: "In Linux, which one is the correct keyword used to end an if statement?",
```

File: src/data/linux-quiz.ts [hunk 4]
```diff
@@ -1269,14 +1269,24 @@ const linuxQuiz = [
...
+    Answer: "fi keyword",
+    Distractor1: "end keyword",
+    Distractor2: "{} p",
+    Distractor3: "end if",
+    Explanation: "fi keyword is used to end an if statement in Linux.",
+    Link: "https://www.geeksforgeeks.org/conditional-statements-shell-script/"
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1289]: We only accept freeCodeCamp resources or official documentation

File: src/data/linux-quiz.ts [hunk 5]
```diff
@@ -1269,14 +1269,24 @@ const linuxQuiz = [
...
       "> greater than operator is used to create a new file in Linux.",
     Link: "https://www.freecodecamp.org/news/learn-the-50-most-used-linux-terminal-commands/"
+  },
+  {
+    Question: "In Linux, How to end an if statement?",
+    Answer: "fi keyword",
+    Distractor1: "end keyword",
+    Distractor2: "{} p",
+    Distractor3: "end if",
+    Explanation: "fi keyword is used to end an if statement in Linux.",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1288]:
```suggestion
    Explanation: "The fi keyword is used to end an if statement in Linux.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/linux-quiz.ts (hunk 4 above)
Comment (jdwilkin4) [reply src/data/linux-quiz.ts:1289]: This link still needs to be changed

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

**Review by Shaik-mohd-huzaifa — COMMENTED** [review]
Check if the link is Changed or not

**Comment by jdwilkin4** [comment]
It looks like you ran into some issues when you tried to resolve a merge conflict because it looks like you deleted both of your questions here.

If you click on the Files Change tab for the PR, it will show you the current state for the PR.
Screen Shot 2022-11-14 at 11 06 04 PM

Screen Shot 2022-11-14 at 11 06 24 PM

As maintainer, I am able to go in and undo these deletions.
But in the future, if you are curious about the current state of your PR, you can view all changed files in the Files Changed tab 👍

**Review by jdwilkin4 — APPROVED** [review]

**Pull request merged**